content/en/user-guide/aws/cost-explorer/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Cost Explorer"
+linkTitle: "Cost Explorer"
+description: >
+  Get started with Cost Explorer on LocalStack
+---
+
+## Introduction
+
+Cost Explorer is a service provided by Amazon Web Services (AWS) that enables you to visualize, analyze, and manage your AWS spending and usage. Cost Explorer offers options to filter and group data by dimensions such as service, region, instance type, and more. With Cost Explorer, you can forecast costs, track budget progress, and set up alerts to receive notifications when spending exceeds predefined thresholds.
+
+LocalStack supports Cost Explorer via the Pro/Team offering, allowing you to use the Cost Explorer APIs in your local environment to create and manage cost category definition, cost anomaly monitors & subscriptions. The supported APIs are available on our [API coverage page](https://docs.localstack.cloud/references/coverage/coverage_ce/), which provides information on the extent of Cost Explorer's integration with LocalStack.
+
+{{< alert title="Note">}}
+LocalStack's Cost Explorer implementation cannot programmatically query your cost and usage data, or provide aggregated data such as total monthly costs or total daily usage. However, you can use the integrations to mock the Cost Explorer APIs and test your workflow locally.
+{{< /alert >}}
+
+## Getting started
+
+This guide is designed for users new to Cost Explorer and assumes basic knowledge of the AWS CLI and our [`awslocal`](https://github.com/localstack/awscli-local) wrapper script.
+
+Start your LocalStack container using your preferred method. We will demonstrate how to mock the Cost Explorer APIs with the AWS CLI.
+
+### Create a Cost Category definition
+
+You can create a Cost Category definition using the [`CreateCostCategoryDefinition`](https://docs.aws.amazon.com/aws-cost-management/latest/APIReference/API_CreateCostCategoryDefinition.html)) API. The following example creates a Cost Category definition using an empty rule condition of type "REGULAR":
+
+{{ <command>}}
+$ awslocal ce create-cost-category-definition \
+    --name test \
```

Comment:
small nit: would move either all parameter flags to a new line, or move the `--name` parameter up a line, for more terse alignment:
```suggestion
$ awslocal ce create-cost-category-definition --name test \
```